3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,146 @@
+console.log("-----------------------------When will we be there??-----------------------------")
+const travelInformation = {
+    speedT: 50, //km/h
+    speed: 50 / 60, //km/min  ====>  speed = 5/6 km/min
+    destinationDistance: 432, //min
+  };
+
+function timeCalculation(speed , displacement){
+    travelTimeInMinute = 432 * 6 / 5 ; //min
+    const houres = Math.floor (518.4 / 60) ;
+    const minutes = travelTimeInMinute - (houres * 60);
+    return `this journey will take ${houres} houres and ${minutes} minutes to arrive to the destination.`
+}
+
+  const travelTime = timeCalculation(5/6 , 432);
+  console.log(travelTime);
+
+
+
+
+console.log("-----------------------------Series duration of my life-----------------------------")
+   const information = {
+    avrageAge: 80, //years
+    year: 365, //days
+    day: 24, //hours
+   }
+   const ageAvrageInHoures= 80 * 365 * 24; //700800 hours aged.
```

Comment:
Should not it be also multiply on 60? Since we have also minutes in our lifes :)